package/MDAnalysis/analysis/electromagnetism.py

```diff
@@ -0,0 +1,59 @@
+from base import AnalysisBase
+import numpy as np
+
+class TotalDipole(AnalysisBase):
+    
+    def __init__(self, trajectory=None, filename='order.dat', selection=None, start=None, stop=None, step=None):
+
+        if selection is None:
+            raise RuntimeError('In class TotalDipole: constroctur requires a selection')
+        else:
+            self.selection        = selection
+
+        self.filename         = filename
+        self.time             = []
```

Comment:
two last lines should be 
(reduced space)

```python
self.filename = filename
self.time = []
```

and so on for other lines (if needed).